docs/testing.md

```diff
@@ -43,7 +49,51 @@ Or if you want to run bats from a different repository or branch, edit `centos7-
         bats_forklift_repo: https://github.com/<YOUR_NAME>/forklift.git
         bats_forklift_version: your-branch
 
-## Pipeline Testing
+### To run Bats on an existing VM for development
+
+1. Clone forklift onto your VM (it's recommended to clone into a directory named other than `forklift` so you don't get mixed up):
+```
+git clone git@github.com:theforeman/forklift.git forklift-devel
+```
+2. Clone the Bats repo:
+```
+git clone https://github.com/sstephenson/bats.git
+```
+3. Install Bats on your VM in the place where `katello-bats` expects it:
```

Comment:
Please don't encourage `katello-bats`. It should die.

Reply:
What command should I be typing to run Bats then?

Reply:
> bats forklift/bats/mytest.bats

oh just run bats directly?

Reply:
Should I delete `katello-bats` and update the doc?